### ISSUE TITLE
Ran an executable - got python error: no module named 'tkinter'

### ISSUE THREAD

**Issue by codeyourdream** [marcelotduarte/cx_Freeze#680]
My app uses Python 3.8 and PySimpleGUI library (which in turn loads tkinter).
I built an executable for Windows using cxfreeze-quickstart (because it didn't work otherwise - it failed with an error described in #560 )

I ran the executable and got this:

![изображение](https://user-images.githubusercontent.com/26959077/85938413-652cdc00-b937-11ea-9052-b9d14f2d9861.png)

**Comment by marcelotduarte** [comment]
PR #576 solve this and will be in the next release. For now, a workaround to build:

python setup.py build_exe --excludes=Tkinter

**Comment by codeyourdream** [comment]
Thanks. I also found that renaming "Tkinter" folder (inside "lib") to "tkinter" makes it work.

**Comment by marcelotduarte** [comment]
Yes, but after each rebuild, you need to rename. The issue can be closed?

**Comment by codeyourdream** [comment]
One more quick question related to importing modules.
I have this module used in my project:

`from hurry.filesize import size`

How can I include into the build? I'm getting the following error:
ImportError: No module named 'hurry.filesize'

**Comment by codeyourdream** [comment]
Also, how can I build the msi (with bdist_msi)? It doesn't seem to work with --excludes=Tkinter

**Comment by marcelotduarte** [comment]
Show me your setup.py

**Comment by codeyourdream** [comment]
I ended up using the default setup.py (generated by cxfreeze-quickstart) and I put the function I needed from hurry.filesize into my code.
Before that I tried various option and nothing worked.
For example, this options list:
`build_exe_options = {"includes": ["hurry.filesize"]}`
or this:
`build_exe_options = {"packages": ["hurry.filesize"]}`

both gave the error mentioned above.

And bdist_msi doesn't support `--excludes` - I'm interested to know how I can build a working installer
